django/contrib/gis/db/backends/postgis/introspection.py

```diff
@@ -24,14 +24,15 @@ class PostGISIntrospection(DatabaseIntrospection):
     # expression over the raster column through the ST_ConvexHull function.
     # So the default query has to be adapted to include raster indices.
     _get_indexes_query = """
-        SELECT DISTINCT attr.attname, idx.indkey, idx.indisunique, idx.indisprimary
-        FROM pg_catalog.pg_class c, pg_catalog.pg_class c2,
-            pg_catalog.pg_index idx, pg_catalog.pg_attribute attr
-        LEFT JOIN pg_catalog.pg_type t ON t.oid = attr.atttypid
```

Comment:
Would like to confirm replacing this `JOIN` by a normal `SELECT` doesn't have any side-effect (although the test cases still pass) since my knowledge about it is limited.